code/model/entity/abstract.php

```diff
@@ -269,10 +269,14 @@ public function setProperty($name, $value, $modified = true)
      */
     public function hasProperty($name)
     {
+        if (empty($name)) {
+            return false;
```

Comment:
Please refactor it anyway. This is part of our coding standards for Kodekit. Don't want to break consistency, if people feel cognitively loaded because of that, that's a bummer.